examples/backup_vm.py

```diff
@@ -499,17 +518,36 @@ def get_backup(connection, backup_service, backup_uuid):
     try:
         backup = backup_service.get()
     except sdk.NotFoundError:
-        last_event = get_backup_events(connection, backup_uuid)[0]
+        last_event = get_last_backup_event(connection, backup_uuid)
         raise RuntimeError("Backup {} does not exist, last reported event: {}"
                            .format(backup_uuid, last_event))
 
     if backup.phase == types.BackupPhase.FAILED:
-        last_event = get_backup_events(connection, backup_uuid)[0]
+        last_event = get_last_backup_event(connection, backup_uuid)
         raise RuntimeError("Backup {} has failed, last reported event: {}"
                            .format(backup_uuid, last_event))
 
     return backup
 
 
+def find_backing_file(backup_dir, checkpoint_uuid, disk_uuid):
+    """
+    Return the name of the backing file for checkpoint, or None if the file was
+    not found.
+
+    Assumes backup filename:
+
+        {timestamp}.{checkpoint-uuid}.{disk-uuid}.{backup-mode}.qcow2
```

Comment:
Weren't we talking at the meeting to use backup ID in the name, not checkpoint ID?

Reply:
I don't remember such discussion, but backup id is not useful for anything but accessing
the current backup entity. The checkpoint id is much more important since it is 
required to start an incremental backup.